### PR TITLE
fix: update editors

### PR DESCRIPTION
**Pull request by cabreraalex**
Fixes a couple and removes shorter and longer - I don't think they made sense, especially stating exact lengths as the fix I don't think makes sense.

### CODE REVIEW

**Comment by Sparkier** [comment]
I will review this after we have projects for all of these, should keep them updated.

**Comment by cabreraalex** [comment]
Okay, will add back! What do you think about the exact length edit?

**Comment by cabreraalex** [comment]
re-added

**Review by Sparkier — APPROVED** [review]

File: src/lib/server/editors/NoNegationEditor.ts [hunk 1]
```diff
@@ -5,48 +5,66 @@ import { EditorType } from '$lib/types';
...
+Input: Don't return free text.
+Output: Always return JSON.
 `;
 
 export class NoNegationEditor extends PromptEditor {
 	constructor() {
 		super(
 			'NoNegation',
-			"Don't Use Negation",
+			'Tell the model what it should do',
```
[the remark this answers is not in the document]
Comment (cabreraalex) [reply src/lib/server/editors/NoNegationEditor.ts:67]: It's technically not negation but I agree it's confusing

**Pull request merged**